src/main/java/securbank/models/Account.java

```diff
@@ -0,0 +1,184 @@
+package securbank.models;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+import org.joda.time.LocalDateTime;
+
+/**
+ * 
+ * @author Madhu Illuri
+ */
+
+@Entity 
+@Table(name = "Account")
+public class Account {
+	
+	/** Account number is unique. */
+	@Id
+	@NotNull
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "accountNumber", nullable = false, unique = true, columnDefinition = "BINARY(16)")
+	private Long accountNumber;
+	
+	/** multiple account can be associated with an user	 */
+	@ManyToOne(cascade = CascadeType.ALL)
```

Comment:
This would be many to many. As a user can have 2 accounts. One for credit card and other is checking